client/src/contexts/CanvasContext.tsx

```diff
@@ -37,7 +39,9 @@ export const CanvasProvider: React.FC<CanvasProviderProps> = ({
   const canvasRef = useRef<HTMLCanvasElement | null>(null);
   const { handleZoom } = useZoom();
   const { socket } = useSocket();
-
+  const { decodedToken } = useAuth();
+  const email = decodedToken?.email ?? "";
+  const color = useMemo(() => getColorFromEmail(email), [email]);
```

Comment:
```suggestion
  const userColor = useMemo(() => getColorFromEmail(email), [email]);
```
Maybe `userColor`? Or`cursorColor`